modular_skyrat/modules/death_consequences_perk/death_consequences_trauma.dm

```diff
@@ -222,28 +225,19 @@
 
 		if (owner.has_reagent(/datum/reagent/medicine/rezadone, needs_metabolizing = TRUE))
 			var/datum/reagent/reagent_instance = owner.reagents.get_reagent(/datum/reagent/medicine/rezadone)
-			if ((reagent_instance.purity >= rezadone_minimum_purity) && reagent_process_flags_valid(owner, reagent_instance))
+			if ((reagent_instance.purity >= DEATH_CONSEQUENCES_REZADONE_MINIMUM_PURITY) && reagent_process_flags_valid(owner, reagent_instance))
 				decrease += rezadone_degradation_decrease
 
-		if (owner.has_reagent(/datum/reagent/medicine/sansufentanyl, needs_metabolizing = TRUE))
-			var/datum/reagent/reagent_instance = owner.reagents.get_reagent(/datum/reagent/medicine/sansufentanyl)
-			if (reagent_process_flags_valid(owner, reagent_instance))
-				decrease += sansufentanyl_degradation_decrease
-
 	if (owner.has_reagent(/datum/reagent/eigenstate))
 		decrease += eigenstasium_degradation_decrease
 
-	if (owner.has_reagent(/datum/reagent/medicine/strange_reagent, needs_metabolizing = TRUE))
-		var/datum/reagent/reagent_instance = owner.reagents.get_reagent(/datum/reagent/medicine/strange_reagent)
-		if (reagent_process_flags_valid(owner, reagent_instance))
-			decrease += strange_reagent_degradation_decrease
-
-	return (decrease * get_passive_degradation_decrease_mult())
+	return (decrease * get_passive_degradation_decrease_resting_mult())
 
 #define DEGRADATION_REDUCTION_SLEEPING_MULT 3
 #define DEGRADATION_REDUCTION_RESTING_MULT 1.5
-/// A global proc used for all scenarios we would decrease passive degradation.
-/datum/brain_trauma/severe/death_consequences/proc/get_passive_degradation_decrease_mult()
+
+/// Returns a multiplier that should be used whenever degradation is passively decreased. Is determined by resting, sleeping, and buckled status.
+/datum/brain_trauma/severe/death_consequences/proc/get_passive_degradation_decrease_resting_mult()
```

Comment:
Use `multiplier`, not `mult`, please. Also, if it's not just from resting, I'd remove the `resting` from the name, too.
```suggestion
/datum/brain_trauma/severe/death_consequences/proc/get_passive_degradation_decrease_resting_multiplier()
```